### PR TITLE
Updated contributing document

### PR DESCRIPTION
**Pull request by StanZGenchev**
Signed-off-by: StanZGenchev <stanislav.genchev@sap.com>

### What does this PR do?

Updates the CONTRIBUTING document, fixing typos and adding new information.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,54 +2,65 @@
...
+
+This project is separated into many smaller projects (git subprojects).
+Those subprojects can be found at the [dirigiblelabs](https://github.com/dirigiblelabs) GitHub page.
+Their names start with 'ide-', 'api-' and 'ext-' depending on what they are for.
+
+If you want to contribute in any way, you must go through the following steps:
+
+1. Fork this repository into your account. Do NOT fork the subproject you want to work on.
+2. Open Dirigible IDE. If you do not have access to a deployed instance, run it locally. See [README](README.md).
+3. From the sidebar, go to the Git section and clone the subproject you will be working on. Use the master/main branch.
```
Comment (delchev) [inline CONTRIBUTING.md:57]: section -> perspective

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,54 +2,65 @@
...
+4. Go to the Editor (the first item in the sidebar) and make your changes. For front-end subprojects, this will automatically apply the changes in real time and make the development process faster.
```
Comment (delchev) [inline CONTRIBUTING.md:58]: Editor -> Workbench

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,54 +2,65 @@
...
 
 ## Coding Style
 
-Please, consider Coding Style for the different languages used in the codebase as 
+Please, consider Coding Style for the different languages used in the codebase as
 described here: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style
+
+## Development process
+
+This project is separated into many smaller projects (git subprojects).
```
Comment (delchev) [inline CONTRIBUTING.md:49]: many smaller projects -> modules

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,54 +2,65 @@
...
+Those subprojects can be found at the [dirigiblelabs](https://github.com/dirigiblelabs) GitHub page.
```
Comment (delchev) [inline CONTRIBUTING.md:50]: Those subprojects -> Some of the modules are standard Java Maven modules and some of them are following the WebJars module structure. The WebJars ones

File: CONTRIBUTING.md (hunk 1 above)
Comment (StanZGenchev) [reply CONTRIBUTING.md]: Fixed all instances of 'section' with 'perspective'.

File: CONTRIBUTING.md (hunk 2 above)
Comment (StanZGenchev) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (StanZGenchev) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 4 above)
Comment (StanZGenchev) [reply CONTRIBUTING.md]: Added the information.

**Pull request merged**